servicemesh-release-notes/topics/rn-new-features.adoc

```diff
@@ -11,13 +11,20 @@ Result – If changed, describe the current user experience
 Starting with {ProductName} 0.9.TechPreview, Mixer’s policy enforcement is disabled by default, but you must enable it to run policy tasks. See https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#update-mixer-policy-enforcement[Update Mixer policy enforcement] for instructions on enabling Mixer policy enforcement.
 ====
 
+= New features Technology Preview 11
+
+The release of {ProductName} adds support for multi-tenant installations, RHEL Universal Base Images (UBI8), OpenSSL 1.1.1, Kiali 0.20.x, the 3scale Istio Adapter 0.6.0, and Istio 1.1.5.
```

Comment:
Unless there's an exception for the UBI, spell out the first instance of RHEL.

Reply:
Fixed.